packages/database/dbSchemas/akash/providerSnapshotStorage.ts

```diff
@@ -0,0 +1,17 @@
+import { DataTypes } from "sequelize";
+import { Column, Default, Model, PrimaryKey, Table } from "sequelize-typescript";
+
+import { Required } from "../decorators/requiredDecorator";
+
+@Table({
+  modelName: "providerSnapshotStorage",
```

Comment:
I would prefer changing naming convention as we move tables to drizzle or do a single PR for renaming everything. Instead of having mixed convention inside our sequelize schemas. wdyt?

Reply:
Ok yea makes sense